site/content/docs/developing/secrets.en.md

```diff
@@ -22,9 +33,19 @@ secrets:
   GITHUB_WEBHOOK_SECRET: GH_WEBHOOK_SECRET  
 ```
 
-Once you deploy this updated manifest, your service or job will be able to access the environment variable `GITHUB_WEBHOOK_SECRET`, which will have the value of the SSM parameter `GH_WEBHOOK_SECRET`, `secretvalue1234`.
-
+Once you deploy this updated manifest, your service or job will be able to access the environment variable `GITHUB_WEBHOOK_SECRET`, which will have the value of the SSM parameter `GH_WEBHOOK_SECRET`, `secretvalue1234`.  
 This works because ECS Agent will resolve the SSM parameter when it starts up your task, and set the environment variable for you.
 
-!!! attention
-    Secrets are not supported for Request-Driven Web Services.
+#### In Secrets Manager
+Similar to SSM, first ensure that your Secrets Manager secret has the `copilot-application` and `copilot-environments` tags.  
+
+Now if you want to pass the Secrets Manager secret named `GH_WEBHOOK_SECRET` with ARN `arn:aws:secretsmanager:us-west-2:111122223333:secret:GH_WEBHOOK_SECRET-WC3PHL`, you can modify your manifest file with:
+```yaml
+secrets:
+  # (Recommended) Option 1. Referring to the secret by name.
+  GITHUB_WEBHOOK_SECRET:
+    secretsmanager: GH_WEBHOOK_SECRET
+
+  # Option 2. Referring to the secret by ARN.
+  GITHUB_WEBHOOK_SECRET: '"arn:aws:secretsmanager:us-west-2:111122223333:secret:GH_WEBHOOK_SECRET-WC3PHL"'
```

Comment:
Apologies for the uninvited review, but it's worth calling out that a very common usage of secrets manager is to store database credentials that have key-value pair in it. It'd be very helpful for the users to see an example that shows how to use a key stored within a secret.

For example, today, my manifest contains this:
```yaml
environments:
  production:
    secrets:
      DB_ENGINE: "'arn:aws:secretsmanager:us-east-1:111122223333:secret:${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/postgres-ABCDEF:engine::'"
      DB_USERNAME: "'arn:aws:secretsmanager:us-east-1:111122223333:secret:${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/postgres-ABCDEF:username::'"
      DB_PASSWORD: "'arn:aws:secretsmanager:us-east-1:111122223333:secret:${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/postgres-ABCDEF:password::'"
      DB_HOST: "'arn:aws:secretsmanager:us-east-1:111122223333:secret:${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/postgres-ABCDEF:host::'"
      DB_PORT: "'arn:aws:secretsmanager:us-east-1:111122223333:secret:${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/postgres-ABCDEF:port::'"
      DB_NAME: "'arn:aws:secretsmanager:us-east-1:111122223333:secret:${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/postgres-ABCDEF:dbname::'"
  staging:
    secrets:
      DB_ENGINE: "'arn:aws:secretsmanager:us-east-1:111122223333:secret:${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/postgres-GHIJKL:engine::'"
      DB_USERNAME: "'arn:aws:secretsmanager:us-east-1:111122223333:secret:${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/postgres-GHIJKL:username::'"
      DB_PASSWORD: "'arn:aws:secretsmanager:us-east-1:111122223333:secret:${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/postgres-GHIJKL:password::'"
      DB_HOST: "'arn:aws:secretsmanager:us-east-1:111122223333:secret:${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/postgres-GHIJKL:host::'"
      DB_PORT: "'arn:aws:secretsmanager:us-east-1:111122223333:secret:${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/postgres-GHIJKL:port::'"
      DB_NAME: "'arn:aws:secretsmanager:us-east-1:111122223333:secret:${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/postgres-GHIJKL:dbname::'"
```

After this change, it would be helpful to know whether it'd look something like this in the root `secrets` section or something else:
```yaml
secrets:
  DB_ENGINE: "${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/postgres:engine::"
  DB_USERNAME: "${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/postgres:username::"
  DB_PASSWORD: "${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/postgres:password::"
  DB_HOST: "${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/postgres:host::"
  DB_PORT: "${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/postgres:port::"
  DB_NAME: "${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/postgres:dbname::"
```

Reply:
This is great feedback, thank you @raxityo !

Indeed, after this feature you'll be able to update the manifest like you described it. I tested the following and it works:
```yaml
secrets:
  DB_ENGINE:
    secretsmanager: '${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/deleteme2:engine::'
  DB_USERNAME:
    secretsmanager: '${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/deleteme2:username::'
  DB_PASSWORD:
    secretsmanager: '${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/deleteme2:password::'
```

I've updated the docs to reflect your suggestions